src/Swarm/TUI/View.hs

```diff
@@ -382,8 +388,10 @@ drawRecipe e inv (Recipe ins outs reqs) =
         , joinableBorder (Edges True False True True)
         , hLimit 2 hBorder
         ]
-  inLen = length ins
+  inLen = length ins + length times
   outLen = length outs
+  times = [(fromIntegral time, timeE) | time /= 1]
+  timeE = mkEntity (defaultEntityDisplay '.') "tick" [] []
```

Comment:
I wanted to add time to the recipe, so I created an ad-hoc entity.

It would be funnier to use a _real entity_, say "grain of time", which would actually be obtainable in the end game and used to build the time machine for `throw`. :hourglass_flowing_sand:

Reply:
It does not look that great in requirements:

![Screenshot from 2021-10-09 00-56-06](https://user-images.githubusercontent.com/44544735/136701161-d7b000ee-bbd2-4950-be0c-3d00067256ef.png)

It makes some sense for it to be in inputs (it *takes time* to finish) so I will just make it look a little nicer and be done with it.